api/src/webhooks/v1/heroku/onStagingRelease.ts

```diff
@@ -0,0 +1,38 @@
+import * as Hapi from '@hapi/hapi';
+import * as Joi from '@hapi/joi';
+
+
+const routes = [
+
+  {
+    method: 'POST',
+    path: '/heroku/on-staging-release',
```

Comment:
If this is in a `v1` folder (i.e. we're versioning the webhooks), we should probably add some versioning to the URL too.

Reply:
That's done when the routes are registered, same as the api. Routes are all prefixed with `/webhooks/v1`